sdk/nodejs/lambda/lambdaMixins.ts

```diff
@@ -317,7 +317,7 @@ export class CallbackFunction<E, R> extends LambdaFunction {
             ...args,
             code: new pulumi.asset.AssetArchive(codePaths),
             handler: serializedFileNameNoExtension + "." + handlerName,
-            runtime: args.runtime || runtime.NodeJS8d10Runtime,
+            runtime: args.runtime || runtime.NodeJS10dXRuntime,
```

Comment:
If we want to make this not a breaking change for existing deployments, we could conditionally add `ignoreChanges: [“runtime”]` only if `!args.runtime`.  I feel like that’s safer and doesn’t have any obvious downsides.

The one downside is that users with existing lambdas would have to set runtime explicitly or force replacement of lambda to pick up new default. The former unfortunately isn’t easy for “magic” lambdas. Maybe that’s enough to make the breaking change preferable over my suggestion?

Curious @cyrusnajmabadi’s thoughts as well.

Reply:
> The one downside is that users with existing lambdas would have to set runtime explicitly 

That doesn't seem like a downside to me :)  It make perfect sense.

1. new lambdas get v10 by default.
2. existing lambdas opt in to v10 when they want it.